specificatie/openapi.yaml

```diff
@@ -648,6 +828,94 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoekobject om te zoeken op geometrie.'
+      type: object
+      properties:      
+        geometrie:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
+        spatialOperator:
+          type: string
+          description: 'Een uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1'
+          enum:
+#            - bbox
+#            - equals
+#            - disjoint
+            - intersects
+#            - touches
+#            - crosses
+            - within
+            - contains
+#            - overlaps
+#            - beyond
+#            - dwithin
+        tolerantie:
+          type: number
+
+    AdressenZoekObject:
+      type: object
+      description: Een object waarin de benodigde zoek criteria worden opgegeven om naar adressen te zoeken.
+      properties:
+        _geo:
```

Comment:
waarom heet dit _geo? En niet gewoon "geometrie"?

Reply:
waarom niet platgeslagen (dus niet in een groep)?

Reply:
@CathyDingemanse we gaan nu adressen zoeken. is het dan niet ook wenselijk de geometrie (uit bijbehorende adresseerbaar objecten) bij de adressen terug te geven, zodat de gebruiker de zoekresultaten direct op een kaart kan tonen? Dus om geometrie in de adressen resource toe te voegen?

Reply:
_geo is (net als _zoek) opvolging van wat er in https://docs.geostandaarden.nl/api/API-Strategie-ext/#call-requests staat.

Reply:
ik zie dit alleen in een voorbeeld (For example, a GEO query to all panden where the geometry in the field _geo ...), niet als voorschrift. of lees ik niet goed?

Reply:
@fsamwel Wat bedoel je met: platgeslagen (dus niet in een groep)? Als het gaat om de GeoJsonZoekOpdracht, die heb ik apart gedefinieerd zodat deze ook gebruikt kan worden bij andere contour zoeken endpoints.

Reply:
platgeslagen zou er zo uit kunnen zien:
```
    AdressenZoekObject:
      allOf:
        - $ref: '#/components/schemas/GeoJsonZoekOpdracht'
        - type: object
          description: Een object waarin de benodigde zoek criteria worden opgegeven om naar adressen te zoeken.
          properties:
            bouwjaar:
              $ref: '#/components/schemas/MinMax'
```

Dus in een request body:
```
{
  "geometrie": {
    "type": "Polygon",
    "coordinates": [
                [
                    118085.231,
                    404358.77
                ],
                [
                    118078.681,
                    404349.51
                ],
                [
                    118134.011,
                    404310.03
                ],
                [
                    118134.657,
                    404310.933
                ]
    ]
  },
  "spatialOperator": "intersects",
  "tolerantie": 0,
  "bouwjaar": {
    "min": 9999,
    "max": 999999
  }
}
```

Reply:
Aangepast.